### DISCUSSION TITLE
Why is NumberInput's text boldness so thin?

### DISCUSSION THREAD

**Discussion by hslee2008** [carbon-design-system/carbon-components-svelte#1722] [Q&A] [answered]
![Screenshot_20230409_133443_Chrome](https://user-images.githubusercontent.com/78584173/230754593-28364d3c-8ec5-47a5-a5ae-765eca7846bc.jpg)
![Screenshot_20230409_133843_Brave](https://user-images.githubusercontent.com/78584173/230754641-b9062e6d-25c8-4184-92b1-266a3c517084.jpg)

I'm not exactly sure whether it is the color or the boldness, but sometimes they are a bit hard to read.

**Comment by theetrain** [comment, marked as answer]
Hello, I checked with the flagship component and it turns out Carbon v11 uses "IBM Plex Sans" whereas Carbon v10 (the styles currently applied to `carbon-components-svelte`) uses "IBM Plex Mono".

![image](https://user-images.githubusercontent.com/12798751/230778743-af657ca3-0d0f-4179-a46c-3c8e267414db.png)

This will eventually get resolved once we adopt v11 Styles; tracked in https://github.com/carbon-design-system/carbon-components-svelte/issues/1629.
